target/linux/ath79/dts/ar9342_avm_fritz3390-wasp.dts

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
I don't know if you are aware that the dts for Friztzbox 3390 lacks a port connecting the two SoCs.

Reply:
Nope, I will add a commit to change that.

Reply:
I have added a patch to add port 5 for 3390 and fix that two ports are not working as reported in the [forum](https://forum.openwrt.org/t/fritzbox-3390-pppoe-issue/97734/4?u=kestrel).

Reply:
Nice. As far as I know a similar error occurs in the 736x family:
https://github.com/openwrt/openwrt/issues/8843
https://github.com/openwrt/openwrt/issues/8833
https://github.com/openwrt/openwrt/issues/5064
https://github.com/openwrt/openwrt/issues/9275

Reply:
@nicefile @skleeschulte You may be interested in this as an owner of one of these devices.

Reply:
Mmh, I wonder if my finding when [using gswip mdio is related](https://github.com/openwrt/openwrt/pull/4939#issuecomment-1037371283)?

Furthermore, if you look into the freetz-ng project kernel source dtbs (arch/mips/boot/dts) it says for hw 202 (7362) and 203 (7362 SL) that extphy1_reset is 45 and extphy2 is 44. So the 37 might be wrong, but there is a 7360 dts which has 37 and 44 as well as hw 183 (7360) and hw 196 (7360 v2 (32MB)). So it seems that the 62 models need their own dts.

Reply:
@abajk I sold all my AVM devices months ago as I've lost interest in these devices.